_posts/2024-06-18-trino-fest-recap.md

```diff
@@ -0,0 +1,121 @@
+cases with Trino both large and small.
+<!--more-->
+
+## Re
+tbd
+
+
+## Sessions
+
+* [What’s new in Trino this summer](https://www.youtube.com/watch?v=mk3n0_tAdZY){:target="_blank"}
+  presented by Martin Traverso from
+  [Starburst](https://www.starburst.io){:target="_blank"}.
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)
+* [Reducing query cost and query runtimes of Trino powered analytics platforms](https://www.youtube.com/watch?v=11yhPXIXiBY){:target="_blank"}
+  presented by Jonas Irgens Kylling from
+  [Dune](){:target="_blank"}.
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)
+* [Enhancing Trino's query performance and data management with Hudi: innovations and future](https://www.youtube.com/watch?v=JMzS2BbeK0E){:target="_blank"}
+  presented by Ethan Guo from
+  [Onehouse](){:target="_blank"}.
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)
+* [Trino Engineering @ Microsoft](https://www.youtube.com/watch?v=t7ndqYUhKSA){:target="_blank"}
+  presented by George Fisher and Ishan Patwa from
+  [Microsoft](){:target="_blank"}.
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)
+* [Enhancing data governance in Trino with the OpenLineage integration](https://www.youtube.com/watch?v=A7hj1M7IYj8){:target="_blank"}
+  presented by Alok Kumar Prusty from
+  [Apple](){:target="_blank"}.
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)
+* [Best practices and insights when migrating to Apache Iceberg for data engineers](https://www.youtube.com/watch?v=dKQ2zShNlyQ){:target="_blank"}
+  presented by Amit Gilad from
+  [Cloudinary](){:target="_blank"}.
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)
+* [Trino query intelligence: insights, recommendations, and predictions](https://www.youtube.com/watch?v=K3iSXOJNaSQ){:target="_blank"}
+  presented by Marton Bod from
+  [Apple](){:target="_blank"}.
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)
+* [The open source journey of the Trino Delta Lake Connector](https://www.youtube.com/watch?v=mPfRYdvDcMo){:target="_blank"}
+  presented by Marius Grama from
+  [Starburst](https://www.starburst.io){:target="_blank"}.
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)
+* [Tiny Trino; new perspectives in small data](https://www.youtube.com/watch?v=ZcY9LJDdB6Y){:target="_blank"}
+  presented by Ben Jeter and Thomas Zugibe from
+  [Executive Homes](){:target="_blank"}.
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)
+* [Bridging the divide: running Trino SQL on a vector data lake powered by Lance](https://www.youtube.com/watch?v=jmOsVbGfon0){:target="_blank"}
+  presented by Lei Xu from LanceDB and Noah Shpak from Character.ai
+  [](){:target="_blank"}.
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)
+* [How FourKites runs a scalable and cost-effective log analytics solution to handle petabytes of logs](https://www.youtube.com/watch?v=xdCZBQJt-0g){:target="_blank"}
```

Comment:
With nearly a petabyte of logs being managed at FourKites, it shouldn't be a huge surprise that they've turned to Trino to handle understanding and analyzing them. Arpit discusses how they've scaled log ingestion, strategically used S3 with Parquet to minimize storage costs, transformed and extracted those logs at scale, and leveraged Trino to search and explore the datasets with Superset as a frontend for visualization.